src/adapters/gumgum.js

```diff
@@ -10,8 +10,16 @@ const GumgumAdapter = function GumgumAdapter() {
 
   const bidEndpoint = `https://g2.gumgum.com/hbid/imp`;
 
-  const WINDOW = global.top;
-  const SCREEN = WINDOW.screen;
+  let WINDOW;
```

Comment:
To keep a clear reference to the variable in a local scope. But I'm open to suggestions. What would be the Prebid approach to this?